CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Master
+
+##### Bug Fixes
+
+* Make linting faster than 0.5.0, but slower than 0.4.0  
+  [Norio Nomura](https://github.com/norio-nomura)
+  [#263](https://github.com/realm/SwiftLint/pull/263)
```

Comment:
You can refer to the SourceKitten issue, you can refer to other changelog entries for the format there.